testing/shunit2/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Olliver Schinagl <oliver@schinagl.nl>
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=shunit2
+pkgver="2.1.7"
+pkgrel=0
+pkgdesc="shUnit2 is a xUnit based unit test framework for Bourne based shell scripts."
+url="https://github.com/kward/shunit2/"
+arch="noarch"
+license="Apache-2.0"
+subpackages="${pkgname}-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/kward/${pkgname}/archive/v${pkgver}.tar.gz"
+
+check()
+{
+	TMPDIR="$(mktemp -d -p "/tmp/" shunit2.XXXXXXXX)" ./gen_test_report.sh
+	rm -rf "/tmp/shunit2.*"
+}
+
+package()
+{
```

Comment:
`package() {`

Reply:
I see both styles used in the code base, and have always used this style on my previous contributions. As a C programmer, it obviously makes more sense :)

I tend to see this style used more in POSIX compliant/portable non-bash shell scripts; but true, it's a mixed bag.

So since this is more a reviewer's choice, and don't see it implemented as a linting rule checker of the test suite/rule written down, I'll leave it as is for now

Reply:
**use** the style as pointed by @tcely

[Thank you](https://github.com/maxice8/atools/blob/master/alint.5.scd#L247)

Reply:
Thank you for your link, but this points to your personal repository. Can you point me to authoritative unbiased information? E.g. meeting minutes or a quote from @ncopa? So far I still see both styles in the repo, and this linter doesn't seem to run during PR's?

Reply:
[Here](https://github.com/alpinelinux/aports/blob/master/CODINGSTYLE.md)

Reply:
you do realize that I instantiated that right? :) So that was kind of a low-blow :p

Reply:
I know that very well, feel free to ping when the changes are done